Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16
+FROM node:16.13.2@sha256:7c49a64aba86dd483aa874bc0230c07f282e20741a7c66e426970ecafc149a38 as base

```

Comment:
We should probably upgrade this to Node v20.7

Reply:
OK, noted. I set the version that was in the NVMRC file, I will update.